Labels and annotations (#3239)
kyma-project/kyma-environment-broker: docs/user/04-40-additional-worker-node-pools.md

    }
    ```

## Labels

> Each additional worker node pool supports an optional **labels** map. Labels are key-value pairs attached to the worker nodes, which you can use to identify, filter, and organize them.

Reviewer: ```suggestion
Each additional worker node pool supports an optional **labels** map. Labels are key-value pairs attached to worker nodes that you can use to identify, filter, and organize them.
```

Each label entry has the following properties:

| Property | Required | Description |
|----------|----------|-------------|
| **key** | Yes | Any non-empty string |

> | **value** | No | Any string |

Reviewer: We use the bold font for actual parameter/key/variable names. From what I see in the example, 'key' and 'value' are more like placeholders for whatever is expressed as a string. Do I understand it correctly?

Author: sure, done.

```bash
curl --request PATCH "https://$BROKER_URL/oauth/v2/service_instances/$INSTANCE_ID?accepts_incomplete=true" \
  --header 'X-Broker-API-Version: 2.14' \
  --header 'Content-Type: application/json' \
  --header "$AUTHORIZATION_HEADER" \
  --data-raw "{
    \"service_id\": \"47c9dcbf-ff30-448e-ab36-d3bad66ba281\",
    \"plan_id\": \"4deee563-e5ec-4731-b9b1-53b42d855f0c\",
    \"context\": {
      \"globalaccount_id\": \"$GLOBAL_ACCOUNT_ID\"
    },
    \"parameters\": {
      \"additionalWorkerNodePools\": [
        {
          \"name\": \"worker-1\",
          \"machineType\": \"Standard_D2s_v5\",
          \"haZones\": true,
          \"autoScalerMin\": 3,
          \"autoScalerMax\": 20,
          \"labels\": {}
        }
      ]
    }
  }"
```

## Annotations

Each additional worker node pool supports an optional **annotations** map. Annotations are key-value pairs used to attach arbitrary non-identifying metadata to worker nodes, such as tooling configuration or operational notes.

Each annotation entry has the following properties:

| Property | Required | Description |
|----------|----------|-------------|
| **key** | Yes | Any non-empty string |

> | **value** | No | Any string |

Reviewer: Please see the comment above.

Author: done.
